lib/hyacinth/adapters/preservation_adapter/fedora3/dc_properties.rb

```diff
@@ -22,7 +22,7 @@ def to(fedora_obj)
         def proposed_dc_properties(hyacinth_obj = @hyacinth_obj)
           properties = {}
           # set the title
-          properties[:title] = [get_title(hyacinth_obj.descriptive_metadata)]
+          properties[:title] = [get_title(hyacinth_obj)]
```

Comment:
Should we be getting the display label instead of the title, in case there is no title?